internal/tiltfile/value/map.go

```diff
@@ -0,0 +1,39 @@
+package value
+
+import (
+	"fmt"
+
+	"go.starlark.net/starlark"
+)
+
+type StringStringMap struct {
```

Comment:
have you considered doing this as 

type StringStringMap map[string]string

?

i think we only need the wrapper struct if we want to treat the "unset" value differently

Reply:
oh, duh